src/sagemaker_tensorflow_container/training.py

```diff
@@ -13,16 +13,127 @@
 
 from __future__ import absolute_import
 
+import json
 import logging
 
 import sagemaker_containers.beta.framework as framework
 
+
 logger = logging.getLogger(__name__)
 
 
+SAGEMAKER_PARAMETER_SERVER_NUM = 'sagemaker_parameter_server_num'
+
+
+def _is_host_master(hosts, current_host):
+    return current_host == hosts[0]
+
+
+def _build_tf_config(hosts, current_host, ps_num=0, ps_task=False):
+    """
+    Builds a dictionary containing cluster information based on number of hosts and number of
```

Comment:
nit: combine the """ with the first line

Reply:
Will do